src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf");
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+}
+
+html {
+  box-sizing: border-box;
+}
```

Comment:
You should add separate @font-face for each font style (one for normal, one for bold). Main font styles set to html tag